src/olympia/devhub/templates/devhub/validate_addon.html

```diff
@@ -42,14 +42,7 @@ <h2>{{ title }}</h2>
   <section id="upload-file" class="validate-addon">
     <div class="list-addon">
       <label>{{ _('Do you want your add-on to be distributed on this site?') }}</label>
-      <div>
-        <label for="{{ new_addon_form.is_unlisted.auto_id }}">
-          {{ new_addon_form.is_unlisted }}
-          {{ new_addon_form.is_unlisted.label }}
-          <span class="tip tooltip"
-                title="{{ new_addon_form.is_unlisted.help_text }}">?</span>
-        </label>
-      </div>
+      {{ new_addon_form.channel }}
```

Comment:
Logically, this page is very similar to `src/olympia/devhub/templates/devhub/addons/submit/distribute.html`. I am not sure whether we  can entirely re-use that template (@diox or @eviljeff might now).

In any case I think you can safely remove `appversion_form` condition. You could also copy and paste some parts of https://github.com/mozilla/addons-server/blob/master/src/olympia/devhub/templates/devhub/addons/submit/distribute.html#L9-L22

Reply:
The template _could_ be re-used, but I think you'd have to extract out the common part into a separate include file and it'd end up being more complicated than just copying and reusing like now.

Reply:
@wagnerand @eviljeff should I leave it as it is then ?

Reply:
IMO, don't try to re-use the template entirely, but there may be some parts you can copy as @wagnerand mentions.

Reply:
@eviljeffI think the only part that can be copied from distribute.html is `<div>
        {{ distribution_form.channel }}
    </div>` So I think this is is a good way in itself.